lib/brexit_data_exporter.rb

```diff
@@ -0,0 +1,176 @@
+require "fileutils"
+
+class BrexitDataExporter
+  def self.call(*args)
+    new(*args).call
+  end
+
+  def initialize(path:)
+    @path = path
+  end
+
+  def call
+    FileUtils::mkdir_p(path)
+
+    organisations.find_each do |organisation|
+      content = content_for(organisation: organisation)
+      next if content.empty?
+
+      puts organisation.title
+
+      slug = organisation.base_path[26..-1] # /government/organisations/
+      organisation_path = File.join(path, "#{slug}.csv")
+
+      export_csv(organisation, organisation_path)
+    end
+  end
+
+  private_class_method :new
+
+private
+
+  CSV_HEADERS = ["Title", "URL", "Last updated", "Document type", "Taxon", "Brexit related?"].freeze
+
+  BREXIT_TAXONS = [
+    "21eee04d-e702-4e7b-9fde-2f6777f1be2c", # business / business and enterprise
+    "ed2ca1f7-5463-4eda-9324-b597e269e242", # business / trade and investment
+    "4ef6d013-ec2d-4ae8-a1a0-5a984e6e6c42", # business / media and communications
+    "da0bc015-f8e5-492c-8d81-6fbf9b18947c", # business / consumer rights and issues
+    "35f5b496-add7-4263-8084-8510461881fe", # business / employing people
+    "ccb77bcc-56b4-419a-b5ce-f7c2234e0546", # business / science and innovation
+    "a1e4659c-dc15-48be-bc4f-6c609ae061dc", # business / uk economy
+    "cb4c8377-6fcb-4022-a7a1-98395eb50508", # business / water industry
+
+    "dd767840-363e-43ad-8835-c9ab516633de", # education / funding and higher education skills and vocational training
+    "ff00b8b2-dcdb-4659-93c2-291c9be354f3", # education / teaching and leadership
+    "23265b25-7ec3-4960-8517-4ff8d4d92cac", # education / funding and finance for students
+
+    "6e85c12f-f52b-41b3-93ad-59e5f19d64f6", # entering and staying in the uk / rights of foreign nationals in the uk
+    "18c7918f-cde5-4e66-b5f4-cd15c33cc1cc", # entering and staying in the uk / travel and identity documents for foreign nationals
+    "c64eb759-93ff-4bfd-86f2-16753fc1e062", # entering and staying in the uk / immigration rules
+    "29480b00-dc4d-49a0-b48c-25dda8569325", # entering and staying in the uk / visas and entry clearance
+    "262e9ea7-90fc-4ac8-8aa7-fc294282a1d4", # entering and staying in the uk / border control
+    "fa13521f-9285-45b0-bd65-4a472a8037e7", # entering and staying in the uk / immigration offences
+    "fef7e737-6f1a-4ef4-b844-aa24b630ad03", # entering and staying in the uk / permanant stay in the uk
+    "08a8a69f-2825-4fe2-a4cf-c83458a5629e", # entering and staying in the uk / refugee asylum and human rights
+
+    "b14a6778-60a2-483a-924c-774d1040cbf4", # environment / commercial fishing and fisheries
+    "52ff5c99-a17b-42c4-a9d7-2cc92cccca39", # environment / food and farming
+    "42fdd045-cadf-474b-b8a6-ca20a77c43bb", # environment / marine
```

Comment:
You're missing environment - Farming and food grants and payments

Reply:
Good catch, thanks, I've added that in now!